src/import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,43 @@
+import AWS from 'aws-sdk';
+import * as csv from 'csv-parser';
+import { corsHeaders } from '../utils/corsHeaders';
+import { BUCKET } from '../utils/constants';
+
+export const importFileParser = event => {
+  console.log('importFileParser ', event);
+  const s3 = new AWS.S3();
+  
+  event.Records.forEach(record => {
+    const s3Stream = s3.getObject({
+      Bucket: BUCKET,
+      Key: record.s3.object.key
+    }).createReadStream();
+
+    s3Stream.pipe(csv())
+      .on('data', data => console.log('data: ', data))
+      .on('error', error => console.error('error: ', error))
+      .on('end', async () => {
+        console.log(`Copy from ${BUCKET}/${record.s3.object.key}`);
+
+        await s3.copyObject({
+          Bucket: BUCKET,
+          CopySource: `${BUCKET}/${record.s3.object.key}`,
+          Key: record.s3.object.key.replace('uploaded', 'parsed')
+        }).promise();
+        
+        console.log(`Copy into ${BUCKET}/${record.s3.object.key.replace('uploaded', 'parsed')}`);
+
+        await s3.deleteObject({
+          Bucket: BUCKET,
+          Key: record.s3.object.key
+        }).promise();
+        
+        console.log(`Delete from ${BUCKET}/${record.s3.object.key}`);
+      })
+  })
+
+  return {
```

Comment:
[minor] This handler consumes event of `S3Event` type, therefore `importFileParser` has S3Handler type
![image](https://user-images.githubusercontent.com/17451948/99433393-827e3900-291e-11eb-90d1-09d20cd7ec5b.png)
![image](https://user-images.githubusercontent.com/17451948/99433422-8c07a100-291e-11eb-8cfa-d924e15add77.png)

It means you shouldn't return anything from here, it simply useless =) There is a mistake in task 5.2